packaging/installer/methods/kickstart.md

```diff
@@ -94,22 +110,25 @@ should not need to use special values for any of these):
 
 ### Connect node to Netdata Cloud during installation
 
-The `kickstart.sh` script accepts additional parameters to automatically [connect](/claim/README.md) your node to Netdata Cloud immediately after installation. 
+The `kickstart.sh` script accepts additional parameters to automatically [connect](https://github.com/netdata/netdata/blob/master/claim/README.md) your node to Netdata Cloud immediately after installation.
 
-> Note: You either need to run the command with root privileges or run it with the user that is running the agent.  More details: [Connect an agent without root privileges](/claim/README.md#connect-an-agent-without-root-privileges) section.
+:::note
+You either need to run the command with root privileges or run it with the user that is running the agent.  
+More details: [Connect an agent without root privileges](https://github.com/netdata/netdata/blob/master/claim/README.md#connect-an-agent-without-root-privileges) section.
+:::
 
-To automatically claim nodes after installation: 
+To automatically claim nodes after installation:
 
 1. Sign in to [Netdata Cloud](https://app.netdata.cloud/sign-in?cloudRoute=/spaces)
-2. Go to the [Spaces management area](https://learn.netdata.cloud/docs/cloud/spaces#manage-spaces)
+2. Go to the [Spaces management area](https://github.com/netdata/netdata/blob/master/docs/cloud/spaces.md#manage-spaces)
```

Comment:
Don't send them there. Nodes tab and then add nodes is more convenient. 

<img width="1918" alt="image" src="https://user-images.githubusercontent.com/43294513/215824480-ccd1412b-56b5-46f1-b928-0dd95379a09b.png">

For their first node, it's actually easier, the CTA is everywhere. Create a new space and you'll see.

Reply:
I haven't edited the info in this section, only links and formatting, let's figure out what we should do with this section in the comment below.